mirascope/core/base/_create.py

```diff
@@ -187,6 +187,7 @@ async def inner_async(
                 output._model = model
                 return output if not output_parser else output_parser(output)
 
+            inner_async.__mirascope_call__ = True  # pyright: ignore [reportAttributeAccessIssue]
```

Comment:
instead of doing this for both sync and async, just set this on the fn (see `fn._model` for reference)

same for all other cases

Reply:
done